deploy/aws/cloudbeat-aws.yml

```diff
@@ -2,9 +2,11 @@ cloudbeat:
   config:
     v1:
       benchmark: cis_aws
+      aws:
+        credentials:
+          access_key_id: ${AWS_ACCESS_KEY_ID:no_access_key_was_set}
```

Comment:
Why do we have these values instead of nil or empty values that will `omitempty` the keys in unmarshal flow?
`no_access_key_was_set`
`no_secret_was_set`

I think this creates a more complex flow where we populate fields with mock values just to fail later on.
I would expect the behavior to be different where we do not populate the fields and just fail on validation that the fields were non-existent or continue the flow without keys in case of anonymous access (which this change blocks since it mocks something invalid).

Obviously, this would require changes to the `config.go` and potentially changes to the `validator` in case we do not want to rely on `config.go`, but as of today all of our validations happen in the `config.go` layer.

Reply:
No real reason just wanted to be aligned with the EKS config file.
I'll revert it to an empty string.